backend/src/appointment/migrations/versions/2024_12_18_1226-b398005a40e7_add_language_to_subscriber_table.py

```diff
@@ -0,0 +1,37 @@
+"""Add language to subscriber table
+
+Revision ID: b398005a40e7
+Revises: d791a3f0e478
+Create Date: 2024-12-18 12:26:56.211080
+
+"""
+from alembic import op
+import sqlalchemy as sa
+from sqlalchemy.orm import Session
+
+from appointment.database import models
+from appointment.defines import FALLBACK_LOCALE
+
+# revision identifiers, used by Alembic.
+revision = 'b398005a40e7'
+down_revision = 'd791a3f0e478'
+branch_labels = None
+depends_on = None
+
+
+def upgrade() -> None:
+    # Add language column to subscribers table
+    op.add_column('subscribers', sa.Column('language', models.encrypted_type(sa.String), index=True))
+
+    # Prefill new column with default value
+    session = Session(op.get_bind())
+    subscribers: list[models.Subscriber] = session.query(models.Subscriber).all()
```

Comment:
You'll have to confirm but having a default value and nullable=False should remove the need for this.

Reply:
I thought so too, in fact I first tried it with default values. But it didn't work, the language column would always stay empty... I thought maybe it's because of the encrypted type? Maybe you have another idea?

Reply:
I switched to db defaults, you can test the branch again, if you like